client/src/pages/reports/Show.js

```diff
@@ -97,7 +97,9 @@ export default class ReportShow extends Page {
 	}
 
 	renderNoPositionAssignedText() {
-		return <p>Notice: This report cannot be submitted because you do not have an assigned position.<br /> -- Please contact your administrator --</p>
+		const supportEmail = dict.lookup('SUPPORT_EMAIL_ADDR')
+		const supportEmailMessage = supportEmail ? `at ${supportEmail}` : ''
+		return <div className="alert alert-warning">Notice: This report cannot be submitted because you do not have an assigned or active position.<br /> -- please contact the support team {supportEmailMessage} --</div>
```

Comment:
I suggest something like `... please contact your organization's super users and request them to assign you to a position. If you are unsure, you can also contact  the support team ${supportEmailMessage}` @dspCarlson

Reply:
My suggestion: You cannot submit a report.  You are not assigned to an advisor position.  Please contact your organization's super users {some magic to identify the superusers} and request that they assign you to an advisor position.

Reply:
the issue here, at least for now, is that we can't identify the superusers until the user is assigned to a position